openmdao/visualization/n2_viewer/tests/test_viewmodeldata.py

```diff
@@ -138,31 +183,117 @@ def test_model_viewer_has_correct_data_from_sqlite(self):
         r.shutdown()
 
         model_viewer_data = _get_viewer_data(self.sqlite_db_filename)
-        print(model_viewer_data['tree'])
 
         # check expected model tree
-        self.assertDictEqual(model_viewer_data['tree'], self.expected_tree)
+        self.check_model_viewer_data(
+            model_viewer_data,
+            self.expected_tree,
+            self.expected_pathnames,
+            self.expected_conns,
+            self.expected_abs2prom,
+            self.expected_declare_partials,
+            self.expected_driver_name,
+            self.expected_design_vars_names,
+            self.expected_responses_names
+        )
+
+
+    def test_model_viewer_has_correct_data_from_optimization_problem(self):
+        """
+        Verify that the correct model structure data exists when stored as compared
+        to the expected structure, using the SellarStateConnection model.
```

Comment:
this docstring is actually incorrect (copy/paste error).. it's the ActuatorDisc in this case

Reply:
fixed. Thanks